lang/py/setup.py

```diff
@@ -25,11 +27,19 @@
 if version_info[:2] <= (2, 5):
     install_requires.append('simplejson >= 2.0.9')
 
+curdir = os.getcwd()
+version_file = ("VERSION.txt" if os.path.isfile("VERSION.txt")
```

Comment:
Can we wrap this in a function? This would allow us to write a test as well :-)

Reply:
Thanks for the review @Fokko! I'll update the PR.

Reply:
Thanks!